src/Chapter05.Tests/Listing05.33.Tests.cs

```
@@ -5,5 +5,180 @@ namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter05.Listing05_33.Tests;
 [TestClass]
 public class ProgramTests
 {
+    [TestMethod]
+    public void Main_NoArgs_ExpectErrors()
+    {
+        string[] args = Array.Empty<string>();
+        string expected = """
+            ERROR:  You must specify the URL and the file name
+            Usage: Downloader.exe <URL> <TargetFileName>
+            """;
 
+        IntelliTect.TestTools.Console.ConsoleAssert.Expect(expected,
+        () =>
+        {
+            Program.Main(args);
+        });
+    }
+
+#if !NET7_0_OR_GREATER
+[TestMethod]
+    public void Main_NoArgs_ExpectFirstNullError()
+    {
+        string[] args = {null! , "Montoya"};
+        string expected = "Value cannot be null. (Parameter \'httpsUrl\')";
+
+        IntelliTect.TestTools.Console.ConsoleAssert.Expect(expected,
+        () =>
+        {
+            try
+            {
+                Program.Main(args);
+            }
+            catch (Exception ex)
```

Same comment here, we don't want to suppress the failure cases.